{"Message":"خطا در واکشی اطلاعات ","StackTrace":"   at Pajoohesh.UserManagement.BusinessService.UserService.GetUserQuery(GetUserQuery message) in E:\\PajooheshWorkspace\\TotalSystem\\Pajoohesh.UserManagement\\Pajoohesh.UserManagement.Service\\UserService.cs:line 256\r\n   at Pajoohesh.UserManagement.Agent.UserAgent.GetUserQuery(GetUserQuery message) in E:\\PajooheshWorkspace\\TotalSystem\\Pajoohesh.UserManagement\\Pajoohesh.UserManagement.Agent\\UserAgent.cs:line 43\r\n   at Pajoohesh.UserManagement.Web.Controllers.UserController.GetAllUsers(HttpRequestMessage requestMessage) in E:\\PajooheshWorkspace\\TotalSystem\\Pajoohesh.UserManagement\\Pajoohesh.UserManagement.Web\\Controllers\\UserController.cs:line 388\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()","InnerErrorStruct":{"Message":"Object reference not set to an instance of an object.","StackTrace":"   at Pajoohesh.Framework.Data.EF.Repositories.OrderByHelper.Sort[TEntity,TResult](IQueryable`1 source, Query`1 sortInfoCollection) in E:\\PajooheshWorkspace\\Framework\\Pajoohesh.Framework.Data.EF\\Repositories\\OrderByHelper.cs:line 104\r\n   at Pajoohesh.Framework.Data.EF.Repositories.ListQueryResultHelper.SortAndPagedList[TEntity,TMessage](ListQueryResult`1 source, IQueryable`1 query, Query`1 message) in E:\\PajooheshWorkspace\\Framework\\Pajoohesh.Framework.Data.EF\\Repositories\\OrderByHelper.cs:line 251\r\n   at Pajoohesh.UserManagement.BusinessService.UserService.GetUserQuery(GetUserQuery message) in E:\\PajooheshWorkspace\\TotalSystem\\Pajoohesh.UserManagement\\Pajoohesh.UserManagement.Service\\UserService.cs:line 244","InnerErrorStruct":null}}
